app/jobs/google/search_keyword_job.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Google
+  class SearchKeywordJob < ApplicationJob
+    queue_as :default
+
+    def perform(keyword)
+      search_result = Google::SearchService.new(keyword[:name]).call
+      raise Google::Errors::SearchKeywordError unless search_result
```

Comment:
I would not recommend raising an error in this case. Raising an error is useful when you do not know what to do about the error (e.g. when coding a library and want the user to be noticed). But in Ruby, most of the use cases implement a "return false" or "return nil". 
As we are in a Job that updates a keyword, if we can't get any search results, let's just update the keyword `status` column to `failed` and end the job without raising any error :)

Reply:
Added `status` and `failed` status to `Keyword`. 👍